gh-132011: Fix crash on invalid `CALL_LIST_APPEND` deoptimization (#132018)
python/cpython: Misc/NEWS.d/next/Core_and_Builtins/2025-04-02-17-47-14.gh-issue-132011.dNh64H.rst

> Fix crash on incorrect ``CALL_LIST_APPEND`` deoptimization.

Reviewer: I like to keep technical details out of the changelog. How about something like this?
```suggestion
Fix crash when calling :meth:`list.append` as an unbound method.
```

> Fix crash when calling :meth:`list.append` as an unbound method.

Reviewer: Sorry, my previous suggestion caused a warning. Apparently, `list.append` is undocumented.
```suggestion
Fix crash when calling :meth:`!list.append` as an unbound method.
```

Reviewer 2: list.append() is documented in an unusual way: https://docs.python.org/dev/library/stdtypes.html#mutable-sequence-types
